web_src/less/_repository.less

```diff
@@ -1165,6 +1171,10 @@
           box-shadow: none;
         }
       }
+
+      @media @mediaSm {
+        padding: 1rem 0 !important;
```

Comment:
Why important?
Because other (fomantic?) style classes interfere otherwise?

Reply:
Yes.

Reply:
I'd still like a comment here.
As someone (I don't know anymore _who_) said, it's probably a good idea to force comments explaining `!important`s, simply so that at some point it can be easily determined if they are still needed.